### ISSUE TITLE
Add rc file support

### ISSUE THREAD

**Issue by xakraz** [ipanardian/GoBranch#3]
It would be nice if we could add a `.gobranchrc` file in the project repo for branch naming enforcement.

**Comment by ipanardian** [comment]
Yes, I've thought about it. Next I'll add the file configuration for branch naming enforcement. Thanks

**Comment by xakraz** [comment]
I am a complete newbie on go for now, but tell me if I can help somewhere
